plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -169,13 +180,28 @@ private Plugin parsePluginLine(String pluginLine) {
         String pluginName = pluginInfo[0];
         String pluginVersion = "latest";
         String pluginUrl = null;
-        if (pluginInfo.length >= 2) {
-            pluginVersion = pluginInfo[1];
-        }
-        if (pluginInfo.length == 3) {
-            pluginUrl = pluginInfo[2];
+
+        //will default to will default to "http,https,ftp" being valid
+        UrlValidator urlValidator = new UrlValidator();
+
+        if (pluginInfo.length == 2) {
```

Comment:
...and then using streams this can be part of a `forEach` (because there's no getting around how you need to check 2 different places in an array...